src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationKeyVaultOptions.cs

```diff
@@ -34,5 +37,23 @@ public AzureAppConfigurationKeyVaultOptions Register(SecretClient secretClient)
             SecretClients.Add(secretClient);
             return this;
         }
+
+        /// <summary>
+        /// Sets the callback used to resolve key vault references that have no registered <see cref="SecretClient"/>.
+        /// </summary>
+        /// <param name="secretResolver">A callback which returns a value for the given Key Vault reference.</param>
```

Comment:
>A callback which returns a value for the given Key Vault reference. [](start = 41, length = 67)

We can also include details on what the return value `ValueTask<string>` for the callback means.
Example : `A callback that maps the <see cref="Uri"/> of the secret with its value`